aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -486,7 +454,22 @@ ASP.NET Core apps are configured using other configuration providers. For more i
 
 ## Application Pools
 
-When hosting multiple websites on a server, we recommend isolating the apps from each other by running each app in its own app pool. The IIS **Add Website** dialog defaults to this configuration. When **Site name** is provided, the text is automatically transferred to the **Application pool** textbox. A new app pool is created using the site name when the site is added.
+::: moniker range=">= aspnetcore-2.2"
+
```

Comment:
Now that I look at the break-out on paper, it seems strange (and *I wrote IT.* 🙈).

Would this be better? ...

```
App pool isolation is determined by the hosting model:

* In-process hosting &ndash; Apps are required to run in separate app pools.
* Out-of-process hosting &ndash; We recommend isolating the apps from each other by running each app in its own app pool.
```